packages/fetchye-immutable-cache/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "fetchye-immutable-cache",
+  "version": "1.0.0",
+  "description": "Cache library to use with fetchye.",
+  "license": "Apache-2.0",
+  "main": "lib/index.cjs.js",
+  "module": "lib/index.esm.js",
+  "browser": "lib/index.umd.js",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "clean": "rimraf lib",
+    "prebuild": "yarn clean",
+    "build": "rollup -c"
+  },
+  "keywords": ["fetchye-immutable-cache", "immutable", "fetchye"],
+  "contributors": [
+    "James Singleton <James.Singleton1@aexp.com> https://github.com/JamesSingleton)",
+    "Jimmy King  <Jimmy.King@aexp.com> (https://github.com/10xLaCroixDrinker)",
+    "Jonathan Adshead <Jonathan.Adshead@aexp.com> (https://github.com/JAdshead)",
+    "Michael Tobia <Michael.M.Tobia@aexp.com> (https://github.com/Francois-Esquire)",
+    "Nelly Kiboi <Nelly.J.Kiboi@aexp.com> (https://github.com/nellyk)"
+  ],
+  "devDependencies": {
+    "@rollup/plugin-babel": "^5.2.1",
+    "@rollup/plugin-commonjs": "^15.1.0",
+    "@rollup/plugin-node-resolve": "^9.0.0",
+    "immutable": "^4.0.0-rc.12",
```

Comment:
immutable would be a full dependency and no longer a peer dep. this is the main reason for putting in its own package